mingw-w64-python-texttable/PKGBUILD

```
@@ -0,0 +1,46 @@
+_realname=texttable
+pkgbase=mingw-w64-python-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-python-${_realname}")
+pkgver=1.7.0
+pkgrel=1
+pkgdesc="Python module to create simple ASCII tables (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url='https://pypi.org/project/texttable/'
+license=('LICENSE')
+depends=("${MINGW_PACKAGE_PREFIX}-python")
+makedepends=("${MINGW_PACKAGE_PREFIX}-python-build" "${MINGW_PACKAGE_PREFIX}-python-installer" 
+"$MINGW_PACKAGE_PREFIX-python-pip" "$MINGW_PACKAGE_PREFIX-python-setuptools" "$MINGW_PACKAGE_PREFIX-python-wheel" 
```

```suggestion
makedepends=("${MINGW_PACKAGE_PREFIX}-python-build" "${MINGW_PACKAGE_PREFIX}-python-installer" 
"$MINGW_PACKAGE_PREFIX-python-setuptools" "$MINGW_PACKAGE_PREFIX-python-wheel" 
```

pip can't be a makedep